frontend/src/Pages/EventAttendance.tsx

```diff
@@ -185,8 +193,14 @@ export default function EventAttendance() {
             return newMods;
         });
     }
-
     async function onSubmit() {
+        if (!canEditAttendance) {
+            toaster(
+                'Cannot update attendance for completed or canceled classes',
```

Comment:
@corypride Our version of spelling is "cancelled" (I've been burned by it about 5 times)